packages/node/src/Session.spec.ts

```diff
@@ -316,6 +316,31 @@ describe("Session", () => {
         mySession.handleIncomingRedirect("https://some.url")
       ).resolves.not.toThrow();
     });
+
+    it("sets the appropriate information before calling the callback", async (done) => {
```

Comment:
I'm not sure if this check is just triggered by the name "callback" in the description, but best to deal with it before merging in.

![image](https://user-images.githubusercontent.com/4251/104913846-836b2f80-598e-11eb-9165-3763e3a85205.png)

Reply:
80a6e1d11ae1bff9b4437654f60b9193209820e0